graylog2-web-interface/src/views/components/messagelist/MessageFields/MessageFieldsEditModeList.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import React, { useCallback, useContext } from 'react';
+import styled, { css } from 'styled-components';
+
+import MessageFieldEditMode from 'views/components/messagelist/MessageFields/MessageFieldEditMode';
+import type { MessageFieldsListProps, FormattedField } from 'views/components/messagelist/MessageFields/types';
+import { SortableList } from 'components/common';
+import MessageFavoriteFieldsContext from 'views/components/contexts/MessageFavoriteFieldsContext';
+
+const Container = styled.div(
+  ({ theme }) => css`
+    margin-left: ${theme.spacings.sm};
+    display: flex;
+    flex-direction: column;
+    gap: ${theme.spacings.xs};
+    padding: ${theme.spacings.xs} 0;
+  `,
+);
+
+const MessageFieldsEditModeList = ({ fields, message, isFavorite }: MessageFieldsListProps) => {
+  const { setFavorites } = useContext(MessageFavoriteFieldsContext);
+
+  const customContentRender = useCallback(
+    ({ item: { field, value, type } }) => (
+      <MessageFieldEditMode
+        key={field}
+        fieldName={field}
+        fieldType={type}
+        message={message}
+        value={value}
+        isFavorite={isFavorite}
+      />
+    ),
+    [isFavorite, message],
+  );
+
+  const onMoveItem = useCallback(
+    (items: Array<FormattedField>) => {
+      setFavorites(items.map((item: FormattedField) => item.field));
+    },
+    [setFavorites],
+  );
+
+  if (!fields.length) return null;
+
+  return (
+    <>
+      <h6>{isFavorite ? 'Favorites' : 'Details'}</h6>
```

Comment:
The modal headline is an `h2`.  Since the heading levels are ment to represent the document’s outline structure, not just the visual size, we should use an `h3` here.